openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -130,6 +130,10 @@ class SiteConfigurationHistory(TimeStampedModel):
         load_kwargs={'object_pairs_hook': collections.OrderedDict}
     )
 
+    class Meta:
+        get_latest_by = 'modified'
+        ordering = ('-modified', '-created',)
```

Comment:
Probably not an issue here, but worth noting that those cols don't seem to be indexed. Only 130 odd rows in the read replica (and 285 for ApiAccessRequest), but thought I'd bring it up. I also happened to see an "api_admin_historicalapiraccessequest" table in the read replica that I think is an orphan?

Reply:
These got silently added to the model definition by the previous implementation of TimeStampedModel, and I didn't want to risk breaking something by removing it at this time.  Also, removing it now would require a new migration (although one that shouldn't actually perform any database changes).  It might merit a separate ticket to review all these and see if it's worth adding indexes or removing the default sorting, though.

The orphaned table is one of the ones that should (eventually) get cleaned up by [PLAT-1756](https://openedx.atlassian.net/browse/PLAT-1756).